pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -200,19 +183,6 @@ a img {
     margin: 0 auto;
 }
 
-/* List rules */
-ul,
-ol {
-    padding: 0;
-    margin: 0 0 0 1.5rem;
-}
-ul {list-style: square}
-ol.arabic {list-style: decimal}
-ol.loweralpha {list-style: lower-alpha}
-ol.upperalpha {list-style: upper-alpha}
-ol.lowerroman {list-style: lower-roman}
-ol.upperroman {list-style: upper-roman}
```

Comment:
These are required for e.g. https://hugovk-peps.readthedocs.io/en/sfs/pep-0209/ -- note that the second list now does not adhere to the source text.

Reply:
Ah yes, reverted those four.

Original:

<img width="302" alt="image" src="https://user-images.githubusercontent.com/1324225/235455005-95d12f60-00e2-43f0-8c07-4927dfe0032d.png">

https://peps.python.org/pep-0209/#proposal

PR before:

<img width="377" alt="image" src="https://user-images.githubusercontent.com/1324225/235455098-0cb9abce-8f3c-4cc6-b2b7-ba36b7003c62.png">

PR now:

<img width="361" alt="image" src="https://user-images.githubusercontent.com/1324225/235455195-9785d137-633a-4fb2-ab14-bd4604b19401.png">

https://peps.python.org/pep-0209/#proposal